index.html

```diff
@@ -0,0 +1,81 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400&display=swap" rel="stylesheet">
+
+
+    <link rel="stylesheet" href="style.css">
+
+
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div class="status-bar">
+            <div class="time-section">
+                <a href="#">
+                    <h2>9:41</h2>
+                </a>
+            </div>
+            <div class="right-section">
+                <img src="img/CellularConnection.png" alt="cellularconnection">
+                <img src="img/Wifi.png" alt="wifi">
+                <img src="img/Battery.png" alt="battery">
+            </div>
+        </div>
+        <div class="toolbar">
+            <div class="logo-section">
+                <h2>My Logo</h2>
+            </div>
+            <div class="icon-menu">
+                <img src="img/Union.png" alt="union">
+            </div>
+        </div>
+    </header>
+    <div class="body-section">
+        <div class="primary-text">
+            <h2>I'm Daniel</h2>
+            <h2>Glad to see</h2>
+            <h2>you!</h2>
+        </div>
+        <div class="secondary-text">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="connect">
+            <h2>LET'S CONNECT</h2>
+        </div>
+        <div class="social-media-icons">
+            <a href="https://twitter.com/DannyDotcoder" target="_blank" rel="noreferrer noopener">
+                <img src="img/Vector.png" alt="vector">
+            </a>
+            <a href="https://www.linkedin.com/in/daniel-shdeed-832b03115/" target="_blank" rel="noreferrer noopener">
+                <img src="img/Linkedinicon.png" alt="linkedinicon">
+            </a>
+            <a href="#" target="_blank" rel="noreferrer noopener">
+                <img src="img/Vector(1).png" alt="ector1">
+            </a>
+            <a href="https://github.com/Danieldotcomcoder" target="_blank" rel="noreferrer noopener">
+                <img src="img/Vector(2).png" alt="vector2">
+            </a>
+            <a href="#" target="_blank" rel="noreferrer noopener">
+                <img src="img/Vector(3).png" alt="vector3">
+            </a>
+        </div>
+    </div>
```

Comment:
# [OPTIONAL]
- [ ] Please consider changing your `div` tag in this icons list for a `ul` tag and put the icons in their individual `li` tags (ul == unordered list, li == list item) this will make your code more semantic.
![image](https://user-images.githubusercontent.com/73256642/133631130-33f8c3e7-266e-4eac-9eca-cf10fac8a0ea.png)